tests/test_save_load.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import pytest
+
+import cleanvision
+from cleanvision.imagelab import Imagelab
+
+
```

Comment:
add to one of the unit tests that actually runs imagelab:

imagelab.save(file)

imagelab2 = imagelab.load(file)

assert issues, issue_summary, info, and other attributes match between imagelab and imagelab2

Reply:
afaict, none of the current tests actually checks the load recovers the saved object exactly.